etcdserver/security/security.go

```diff
@@ -116,7 +116,6 @@ func NewStore(server doer, timeout time.Duration) *Store {
 		server:  server,
 		timeout: timeout,
 	}
-	s.ensureSecurityDirectories()
 	s.enabled = s.detectSecurity()
```

Comment:
Sure. It could be fetched from the store each time, sans quorum=true -- it'll be pretty fast.